SWIG/cashflows.i

```diff
@@ -616,6 +622,8 @@ Leg _FixedRateLeg(const Schedule& schedule,
         .withNotionals(nominals)
         .withCouponRates(couponRates,dayCount)
         .withPaymentAdjustment(paymentAdjustment)
+        .withPaymentCalendar(paymentCalendar)
```

Comment:
This catches a null calendar and replaces it with the default one:
```suggestion
        .withPaymentCalendar(paymentCalendar.empty() ? schedule.calendar() : paymentCalendar)
```

Reply:
Great! Thank you for the suggestion. I'll implemented the changes now.